InvenTree/InvenTree/helpers.py

```diff
@@ -1091,12 +1085,7 @@ def inheritors(cls):
     return subcls
 
 
-class InvenTreeTestCase(ExchangeRateMixin, UserMixin, TestCase):
-    """Testcase with user setup buildin."""
-    pass
-
-
-def notify_responsible(instance, sender, content: NotificationBody = InvenTreeNotificationBodies.NewOrder, exclude=None):
+def notify_responsible(instance, sender, content=None, exclude=None):
```

Comment:
Question: @SchrodingersGat Why are the typings removed?

Reply:
Trying to work out the strange circular import issues for this PR. If you look at the errors, there's some weird stuff going on.